### PR TITLE
feat(onboarding): migration to add `disable_member_invite` bit flag

### PR DESCRIPTION
**Pull request by ameliahsu**
add `disable_member_invite` flag for org admin to allow/disallow members from inviting teammates to their org

frontend: https://github.com/getsentry/sentry/pull/76085

### CODE REVIEW

File: src/sentry/models/organization.py [hunk 1]
```diff
@@ -195,6 +195,9 @@ class flags(TypedClassBitField):
         # Enable codecov integration.
         codecov_access: bool
 
+        # Disable org-members from inviting members
+        disable_member_invite: bool
```
Comment (iamrajjoshi) [inline src/sentry/models/organization.py:199]: we need to make sure a new flag in the bitfield is at the bottom of the field.

File: src/sentry/models/organization.py [hunk 2]
```diff
@@ -201,6 +201,9 @@ class flags(TypedClassBitField):
         # Prevent superuser access to an organization
         prevent_superuser_access: bool
 
+        # Disable org-members from inviting members
+        disable_member_invite: bool
```
Comment (cathteng) [inline src/sentry/models/organization.py:205]: do we want disable to be true or false by default? the default for a bitfield is false

Comment (ameliahsu) [reply src/sentry/models/organization.py:205]: @cathteng by default we want members to invite other members, so we want this flag to be false by default

Comment (sentaur-athena) [reply src/sentry/models/organization.py:205]: yes, default should be false.

Comment (sentaur-athena) [reply src/sentry/models/organization.py:205]: We have a feature flag for launching it gradually but default is false unless they turn it to true to disable.

**Review by wedamija — COMMENTED** [review]
migration lgtm

**Review by iamrajjoshi — APPROVED** [review]

**Pull request merged**